components/brave_rewards/resources/shared/lib/publisher_status.ts

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+export const enum PublisherStatus {
```

Comment:
I'm thinking that we should keep this in the `tip` application for now, as I don't really want all Rewards UI to start using these mojo enum values until we have a better idea of how to surface the C++ API in JS.

Reply:
Actually, I see that we also need this on the rewards page as well, so it's fine to have it here. 👍